dev/run-tests.py

```diff
@@ -23,145 +23,21 @@
 import re
 import sys
 import subprocess
+from utils import *
 
 from sparktestsupport import SPARK_HOME, USER_HOME, ERROR_CODES
 from sparktestsupport.shellutils import exit_from_command_with_retcode, run_cmd, rm_r, which
 from sparktestsupport.toposort import toposort_flatten
 import sparktestsupport.modules as modules
 
 
-# -------------------------------------------------------------------------------------------------
```

Comment:
One thing is that the doctests in `dev/utils.py ` wouldn't be executed (because the script executes the test before each invokation - https://github.com/apache/spark/blob/ea14cb92a146bcd6fa208e9395da1051c037e64a/dev/run-tests.py#L631)